portfolio/src/main/webapp/template.html

```diff
@@ -154,44 +155,41 @@ <h2 class="heading sr-only">Basic Information</h2>
                     <div class="section-inner shadow-sm rounded">
                         <h2 class="heading">Skills</h2>
                         <div class="content">
-                            <p class="intro">
-                                Intro about your skills goes here. Keep the list lean and only show your primary skillset. You can always provide a link to your Linkedin or Github profile so people can get more info there.</p>
-                            
                             <div class="skillset">
-                               
+
```

Comment:
Nit:
In some spaces you have new lines between the first nested tag and in most places you don't. Pick a strategy and try to be consistent everywhere:
https://google.github.io/styleguide/htmlcssguide.html#Parting_Words